revisions-extended/includes/rest-revisions-controller.php

```diff
@@ -204,7 +204,12 @@ public function create_item( $request ) {
 		$response = $this->prepare_item_for_response( $revision, $request );
 		remove_filter( 'rest_prepare_revision', array( $this, 'filter_rest_prepare_revision' ) );
 
-		return rest_ensure_response( $response );
+		$response = $this->prepare_response_for_collection( $response );
+		$response = rest_ensure_response( $response );
+		$response->set_status( 201 );
+		$response->header( 'Location', rest_url( sprintf( 'wp/v2/revision/%d', $revision_id ) ) );
```

Comment:
@StevenDufresne this might also be a breaking change, but it seems unlikely... ?

Reply:
Tested. Everything seems to be working as expected.